src/Components/Display.jsx

```diff
@@ -1,16 +1,23 @@
-import React, {useState} from 'react';
+import React, { useState } from "react";
 
 function Display(props) {
-    // const [display,setDisplay] = useState(0);
-
-    return (
-        <section className="display">
-            This is the Display for the Counter.
-        </section>
-    )
+  // const [display,setDisplay] = useState(0);
+  const [isOn, setIsOn] = useState(false);
 
+  const toggleLight = () => {
+    setIsOn((prevIsOn) => !prevIsOn);
+  };
 
+  return (
+    <div
+      style={{
+        color: isOn ? "yellow" : "blue",
+      }}
```

Comment:
get in the habit of using external stylesheets for styling.  sometimes, inline styles gets to be a pain to track down to fix styling.